doc/build/core/pooling.rst

```diff
@@ -509,30 +509,32 @@ particular error should be considered a "disconnect" situation or not, as well
 as if this disconnect should cause the entire connection pool to be invalidated
 or not.
 
-For example, to add support to consider the Oracle error codes
-``DPY-1001`` and ``DPY-4011`` to be handled as disconnect codes, apply an
-event handler to the engine after creation::
+For example, to add support to consider the Oracle Database driver error codes
+``DPY-1001`` and ``DPY-4011`` to be handled as disconnect codes, apply an event
+handler to the engine after creation::
 
     import re
 
     from sqlalchemy import create_engine
 
-    engine = create_engine("oracle://scott:tiger@dnsname")
+    engine = create_engine(
+        "oracle+oracledb://scott:tiger@localhost:1521?service_name=freepdb1"
+    )
 
 
     @event.listens_for(engine, "handle_error")
     def handle_exception(context: ExceptionContext) -> None:
         if not context.is_disconnect and re.match(
-            r"^(?:DPI-1001|DPI-4011)", str(context.original_exception)
```

Comment:
**Michael Bayer** (zzzeek) wrote:

oh interesting i wonder why these said DPI here when the paragraph said DPY

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5589